src/features/NoResultsPage/styled.js

```diff
@@ -0,0 +1,28 @@
+import styled from "styled-components";
+
+export const NoResultsText = styled.p`
+  font-weight: 600;
+  font-size: 33px;
+  letter-spacing: -0.5px;
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.large}) {
+    font-size: 30px;
+  }
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.medium}) {
+    font-size: 22px;
+  }
+`;
+
+export const ImageContainer = styled.div`
+  height: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`;
```

Comment:
In my opinion, the ImageContainer is useless. You can add to Image `margin: 0 auto;` and change display to block and it will work the same way 🙂

Reply:
You're right, thanks for the feedback!